tests/installation/scc_registration.pm

```diff
@@ -59,6 +59,14 @@ sub run {
                 bugref       => 'bsc#1028774'
             );
         }
+        if (match_has_tag('registration-online-repos')) {
+            if (is_sle('=12-SP5') && check_var('KEEP_REGISTERED', '1')) {
```

Comment:
Only 12-sp5 we need this?

Reply:
only sle12sp4 to sle12sp5 regression tests have this kind of scenario, we can enlarge the scope
once further tests needed.